app/dashboard/templates/dashboard.html

```diff
@@ -32,12 +32,12 @@
       {% include 'shared/nav.html' %}
     </div>
     <div class="row no-gutter">
-      <div class="col-12 col-lg-2" id="sidebar_container" style="background: #F2F6F9;">
+      <div class="col-12 col-lg-3" id="sidebar_container" style="background: #F2F6F9;">
```

Comment:
@joshmobley why was this the width increased here ?

Reply:
![image](https://user-images.githubusercontent.com/933565/38125074-d5314302-33b4-11e8-8b95-4e54305e1c06.png)

This unfortunate wrapping occurs at the lower end of that breakpoint with a 2-column width. 

In fairness, I suppose we could go to the 2/10 column ratio at the "xl" breakpoint if we want to preserve that ratio on larger screens.

Reply:
Yeah could we throw in the `xl` breakpoint ? Primarily our users are on larger screen and it looks a lil off when the sidebar takes up that much screen on the larger screen ! ^_^